### DISCUSSION TITLE
how do i run commands with rtl_433 in windows terminal?

### DISCUSSION THREAD

**Discussion by virxually** [merbanan/rtl_433#2350] [Q&A] [answered]
rtl_433.exe opens and runs fine in a cmd window, but i cant type any commands, i am specifically trying to switch to a different frequency

**Comment by zuckschwerdt** [comment, marked as answer]
The program is not interactive, all instructions need to be given as arguments. E.g. `rtl_433 -f 868M`

The HTTP UI might be better to browse bands. Add the option `-F http` then open http://127.0.0.1:8433/

To get a view of all transmission on a band now click "Modes" "Raw (all)".

**Reply by ghost** [reply]
I'm a newbie to this command stuff. I run rtl_433 win version. you say run -f 868M but where and how do the newbies run this. Glad this question was ask. but the answer left out alot for people just starting with command line stuff. Thanks David

**Reply by zuckschwerdt** [reply]
The `-f 868M` is just some option to be added. The command you run then is `rtl_433 -f 868M` (like actually shown above) which on Windows is short for `rtl_433.exe -f 868M`
But indeed this is not ergonomic for beginners. We might want to make Web-UI a default. If you want to try that use this command:
`rtl_433 -F http`

**Reply by ghost** [reply]
thanks for the reply, how do I find WEB-UI?  run it etc etc.. using win 7, is there a page that shows step by step how to set this up?

**Reply by zuckschwerdt** [reply]
For the Web-UI open http://127.0.0.1:8433/ in a browser

**Reply by ghost** [reply]
Firefox can’t establish a connection to the server at 127.0.0.1:8433 this is what I get..

**Reply by zuckschwerdt** [reply]
Somebody else wants to walk @virxually through this please?

**Reply by virxually** [reply]
> Firefox can’t establish a connection to the server at 127.0.0.1:8433 this is what I get..

to run a command you have to open windows command prompt, then type "cd (rtl_433 folder directory, not rtl_433.exe)"
example: "cd C:\Users\\(name)\Downloads\\(rtl_433 folder)"
if inputted correctly, the command prompt header should change to your rtl_433 folder directory
then type "rtl_433 (command(s))" and enter, rtl_433 will then start running with the variables you added.
(to be fair, @zuckschwerdt could have done a better job at explaining this) ^

running the -F http command would look like this:
(directory)>rtl_433 -F http
press enter and rtl_433 should successfully launch
when it does, go to http://127.0.0.1:8433/ 
if you haven't already, try clicking the hyperlink above, you might be typing something wrong.
i am also on Firefox, and that IP works fine for me when doing -F http.
you might be on an older version of rtl_433, if not then I'm sorry I can't really tell you what your issue is.
